dhcpv6/options.go

```diff
@@ -57,6 +57,8 @@ func ParseOption(code OptionCode, optData []byte) (Option, error) {
 		opt, err = parseOptElapsedTime(optData)
 	case OptionRelayMsg:
 		opt, err = parseOptRelayMsg(optData)
+	case OptionRelayPort:
```

Comment:
NIT: I think this is sorted by option number

Reply:
:facepalm: yes it is

Reply:
Shouldn't it go almost to the end?

https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/types.go#L224

Reply:
yeah sorry that's what I meant, I hadn't realized it was sorted, I'll update